dftimewolf/lib/processors/turbinia_base.py

```diff
@@ -174,6 +175,48 @@ def _ExtractFiles(self, tgz_path: str, path_to_collect: str) -> str:
     local_path = os.path.join(tempdir, path_to_collect.lstrip('/'))
     return local_path
 
+  def UploadEvidence(self, file_path: Path) -> Optional[str]:
+    """Uploads files to Turbinia via the API server.
+    
+    Args:
+      file_path: Path to the file to be uploaded.
+      
+    Returns:
+      File path to the uploaded file.
+    """
+    path_str = file_path.as_posix()
+    turbinia_evidence_path: str = ''
+    if not file_path.exists():
+      self.ModuleError(f'File {path_str} not found.', critical=True)
+    self.RefreshClientCredentials()
+
+    api_instance = turbinia_evidence_api.TurbiniaEvidenceApi(self.client)
+    self.logger.info(
+        f'Uploading evidence at {path_str} for incident {self.incident_id}'
+    )
+    self.logger.info(f'Incident ID: {self.incident_id}')
+    api_response = api_instance.upload_evidence_with_http_info(
+        [path_str], self.incident_id
+    )
+    if not api_response:
+      self.ModuleError(f'Error uploading file {path_str}', critical=True)
+    data: str = str(api_response.raw_data)
+    try:
+      response = json.loads(data)
+    except json.JSONDecodeError as exception:
+      self.logger.error(f'Error decoding API response: {exception}')
+      return turbinia_evidence_path
+    if not response:
+      self.logger.error('Did not receive a response from the API server.')
+      return turbinia_evidence_path
+    # The API supports multiple file upload, but we're only sending one.
+    file = response[0]
+    turbinia_evidence_path = file.get('file_path')
+    self.logger.info(
+        f'Uploaded {file.get("original_name")} to {turbinia_evidence_path}'
+    )
```

Comment:
Let's change this to `self.PublishMessage` so that it's more visible to the user.

Reply:
Done